rest/model/project.go

```diff
@@ -74,12 +74,13 @@ func (t *APITriggerDefinition) BuildFromService(h interface{}) error {
 }
 
 type APIPatchTriggerDefinition struct {
-	Alias          *string            `json:"alias"`
-	ChildProject   *string            `json:"child_project"`
-	TaskSpecifiers []APITaskSpecifier `json:"task_specifiers"`
-	Status         *string            `json:"status,omitempty"`
-	ParentAsModule *string            `json:"parent_as_module,omitempty"`
-	VariantsTasks  []VariantTask      `json:"variants_tasks,omitempty"`
+	Alias                  *string            `json:"alias"`
+	ChildProject           *string            `json:"child_project"`
```

Comment:
Is this the Id? should we maybe add "Id" to this?

Reply:
We could, but I believe that would introduce a breaking change and would require a little more coordination around releasing since the UI is already requesting `childProject`. I am open to it if you think it's worth doing!

Reply:
hmm. Should we maybe add another field called childProjectId that also holds the id  (and add a quick comment) so that we can eventually make the change next time we touch the UI? I think just calling it childProject might end up being confusing.